src/addgrid.f

```diff
@@ -1,67 +1,64 @@
-!   gribcreate must be called first to initialize a new GRIB2 message.
-!
-! PROGRAM HISTORY LOG:
-! 2000-05-01  Gilbert
-!
-! USAGE:    CALL addgrid(cgrib,lcgrib,igds,igdstmpl,igdstmplen,
-!                        ideflist,idefnum,ierr)
-!   INPUT ARGUMENT LIST:
-!     cgrib    - Character array to contain the GRIB2 message
-!     lcgrib   - Maximum length (bytes) of array cgrib.
-!     igds     - Contains information needed for GRIB Grid Definition Section 3.
-!                Must be dimensioned >= 5.
-!                igds(1)=Source of grid definition (see Code Table 3.0)
-!                igds(2)=Number of grid points in the defined grid.
-!                igds(3)=Number of octets needed for each 
-!                            additional grid points definition.  
-!                            Used to define number of
-!                            points in each row ( or column ) for
-!                            non-regular grids.  
-!                            = 0, if using regular grid.
-!                igds(4)=Interpretation of list for optional points 
-!                            definition.  (Code Table 3.11)
-!                igds(5)=Grid Definition Template Number (Code Table 3.1)
-!     igdstmpl - Contains the data values for the specified Grid Definition
-!                Template ( NN=igds(5) ).  Each element of this integer 
-!                array contains an entry (in the order specified) of Grid
-!                Defintion Template 3.NN
-!   igdstmplen - Max dimension of igdstmpl()
-!     ideflist - (Used if igds(3) .ne. 0)  This array contains the
-!                number of grid points contained in each row ( or column )
-!      idefnum - (Used if igds(3) .ne. 0)  The number of entries
-!                in array ideflist.  i.e. number of rows ( or columns )
-!                for which optional grid points are defined.
-!
-!   OUTPUT ARGUMENT LIST:      
-!     cgrib    - Character array to contain the GRIB2 message
-!     ierr     - Error return code.
-!                0 = no error
-!                1 = GRIB message was not initialized.  Need to call
-!                    routine gribcreate first.
-!                2 = GRIB message already complete.  Cannot add new section.
-!                3 = Sum of Section byte counts doesn't add to total byte count.
-!                4 = Previous Section was not 1, 2 or 7.
-!                5 = Could not find requested Grid Definition Template.
-!
-! REMARKS: Note that the Local Use Section ( Section 2 ) can only follow
-!          Section 1 or Section 7 in a GRIB2 message.
-!
-! ATTRIBUTES:
-!   LANGUAGE: Fortran 90
-!   MACHINE:  IBM SP
-!
-!$$$
+!>    @file
+!>    @brief This subroutine packs up a Grid Definition Section
+!>    (Section 3) and adds it to a GRIB2 message.
+!>    @author Gilbert ORG: W/NP11 @date 2000-05-01
+!>
+!>    This routine is used with routines "gribcreate", "addlocal", "addfield",
+!>    and "gribend" to create a complete GRIB2 message.  Subroutine
```

Comment:
Wherever there is a period followed by two spaces, change it to a period followed by a single space. So this would become:

`and gribend() to create a complete GRIB2 message. Subroutine
`